.github/workflows/release.yml

```diff
@@ -437,13 +437,53 @@ jobs:
         path: |
           signed
 
+# ================================
+#           Validate
+# ================================
+  validate:
+    name: Validate installers
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
```

Comment:
This looks great!

One suggestion: It might make sense to imitate the matrix style of [`monitor-components.yml`](https://github.com/git-for-windows/git/blob/79972f5b45e66439383630844fb843a84ccf5cce/.github/workflows/monitor-components.yml#L23) where the matrix values specify more than a single feature. That way, we could express the `needs` value in terms of a matrix variable, and we could likewise avoid downloading unneeded artifacts, again by using a matrix variable.

Reply:
+1 for the matrix/vector

Reply:
This is great!